db/migrate/20150831150909_create_cards.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCards < ActiveRecord::Migration
+  def change
+    create_table :cards do |t|
+      t.string :original_text
+      t.string :translated_text
+      t.date   :review_date
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.